samples/java-webmvc/generated-docs/index.html

```diff
@@ -1143,6 +1143,46 @@ <h4 id="_request_fields_3"><a class="anchor" href="#_request_fields_3"></a><a cl
 </colgroup>
 <thead>
 <tr>
+<th class="tableblock halign-left valign-top">Parameter</th>
+<th class="tableblock halign-left valign-top">Type</th>
+<th class="tableblock halign-left valign-top">Optional</th>
+<th class="tableblock halign-left valign-top">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td class="tableblock halign-left valign-top"><p class="tableblock">description</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">String</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">true<br>
+EN: false</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Some information about the item.</p>
+<p class="tableblock">DE: Size must be between 2 and 10 inclusive.<br>
```

Comment:
This is a new block

Reply:
This block is also on master https://github.com/ScaCap/spring-auto-restdocs/blob/master/samples/java-webmvc/generated-docs/index.html#L1146 so it's strange that appears in the diff.